templates/index.html

```diff
@@ -95,7 +95,7 @@ <h2 class="subtitle">
             </div>
             <div class="upload-btn file is-dark">
               <label class="file-label">
-                <input class="file-input" type="file" name="file" id="file">
+                <input class="file-input" accept="image/*" type="file" name="file" id="file">
```

Comment:
I think this will only open the file picker which shows only image files by default. But if you select to show all files from the file picker, (in the bottom right corner), then you will be able to submit the other files too. 

Therefore, my suggestion is to add a javascript validation too, which will be done before sending it to the backend. Which I have done in #27.

Thank you

Reply:
Yes, you are correct. I can rebase this PR if needed, after your PR gets merged by @shoumikdey.